thoth-export-server/src/csv/csv_thoth.rs

```diff
@@ -0,0 +1,498 @@
+        format!("{:?}", self.subject_code)
+    }
+}
+
+impl CsvCell<CsvThoth> for WorkFundings {
+    fn csv_cell(&self) -> String {
+        format!(
+            "(\"{}\", \"{}\", \"{}\", \"{}\", \"{}\", \"{}\")",
+            self.funder.funder_name,
+            self.funder
+                .funder_doi
+                .clone()
+                .unwrap_or_else(|| "".to_string()),
+            self.program.clone().unwrap_or_else(|| "".to_string()),
+            self.project_name.clone().unwrap_or_else(|| "".to_string()),
+            self.grant_number.clone().unwrap_or_else(|| "".to_string()),
+            self.jurisdiction.clone().unwrap_or_else(|| "".to_string()),
+        )
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use lazy_static::lazy_static;
+    use std::str::FromStr;
+    use thoth_client::{
+        ContributionType, CurrencyCode, LanguageCode, LanguageRelation, PublicationType,
+        WorkContributionsContributor, WorkImprint, WorkImprintPublisher, WorkStatus, WorkType,
+    };
+    use uuid::Uuid;
+
+    lazy_static! {
+        static ref TEST_WORK: Work = Work {
+            work_id: Uuid::from_str("00000000-0000-0000-AAAA-000000000001").unwrap(),
+            work_status: WorkStatus::ACTIVE,
+            full_title: "Book Title: Book Subtitle".to_string(),
+            title: "Book Title".to_string(),
+            subtitle: Some("Book Subtitle".to_string()),
+            work_type: WorkType::MONOGRAPH,
+            edition: 1,
+            doi: Some("https://doi.org/10.00001/BOOK.0001".to_string()),
+            publication_date: None,
+            license: Some("http://creativecommons.org/licenses/by/4.0/".to_string()),
+            copyright_holder: "Author 1; Author 2".to_string(),
+            short_abstract: Some("Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum vel libero eleifend, ultrices purus vitae, suscipit ligula. Aliquam ornare quam et nulla vestibulum, id euismod tellus malesuada. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.".to_string()),
+            long_abstract: Some("Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum vel libero eleifend, ultrices purus vitae, suscipit ligula. Aliquam ornare quam et nulla vestibulum, id euismod tellus malesuada. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Nullam ornare bibendum ex nec dapibus. Proin porta risus elementum odio feugiat tempus. Etiam eu felis ac metus viverra ornare. In consectetur neque sed feugiat ornare. Mauris at purus fringilla orci tincidunt pulvinar sed a massa. Nullam vestibulum posuere augue, sit amet tincidunt nisl pulvinar ac.".to_string()),
+            general_note: None,
+            place: Some("León, Spain".to_string()),
+            width: Some(156),
+            height: Some(234),
+            page_count: Some(334),
+            page_breakdown: Some("x+334".to_string()),
```

Comment:
This is never output in the result

Reply:
I hadn't noticed the breakdown wasn't included in the CSV... It's included now